### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by Saqib198** [Schweinepriester/github-profile-achievements#917] [General]
What will i do for more badges???

**Comment by matteofuso** [comment]
To get the Galaxy Brain Badge you must

1. Get at least 2 marked answers in public discussion
2. You cannot mark your own responses

You may need more than 2 responses to discussions

**Comment by Fuatorium** [comment]
You can use these link: https://www.youtube.com/watch?v=oJWFfp3sGYA&t=348s&ab_channel=Thinkright20

**Reply by 17351269** [reply]
https://www.youtube.com/watch?v=oJWFfp3sGYA&t=348s&ab_channel=Thinkright20
